src/config-schema.ts

```diff
@@ -59,4 +59,9 @@ export const configSchema = {
     _default: false,
     _description: 'Whether to enable form validation',
   },
+  enableSchemaValidation: {
```

Comment:
This should be changed to something like `blockRenderingWithErrors` in other words, allowing/not allowing to render if there are errors.